plugin/pkg/scheduler/scheduler.go

```diff
@@ -107,6 +107,7 @@ func New(c *Config) *Scheduler {
 
 // Run begins watching and scheduling. It starts a goroutine and returns immediately.
 func (s *Scheduler) Run() {
+	go metrics.ReportBindingRateLimiterSaturation(s.config.BindPodsRateLimiter)
```

Comment:
This is specific to binding rate limiter.
The metrics/ dir in also under scheduler/ .
If you want a generic rate limiter, I doubt that's gonna be large changes, which is beyond initial attempt for scalability testing purpose.
Can we raise another issue and ask metrics related folks to do it?